src/cbc_sdk/cache/lru.py

```diff
@@ -128,18 +149,25 @@ def kill_self(o):
             super(LRUCacheDict.EmptyCacheThread, self).__init__()
 
         def run(self):
+            """Execute the background cleanup."""
             while self.ref():
                 c = self.ref()
                 if c:
                     next_expire = c.cleanup()
                     if (next_expire is None):
                         time.sleep(self.peek_duration)
                     else:
-                        time.sleep(next_expire+1)
+                        time.sleep(next_expire + 1)
                 c = None
 
     @_lock_decorator
     def size(self):
+        """
+        Returns the number of values in the dictionary.
+
+        Returns:
```

Comment:
Same comment as `cbc_sdk.base`, if the `Returns` section doesn't provide new info let's remove it

Reply:
OK